### DISCUSSION TITLE
Opening WindowBuilder on one file completely hangs Eclipse

### DISCUSSION THREAD

**Discussion by jsculley** [eclipse/windowbuilder#304] [Q&A]
Eclipse 2022-06
JDK 18
WB 1.98 and 1.99.pre are both listed as Installed according to Eclipse Marketplace

I was trying out WB on an existing Swing codebase.  At some point it stopped working completely when I tried to open a particular file in WB.  A Java frame opened with no title and no content (just the Duke window icon) and Eclipse was completely hung at this point.  Other files open in WB with no problem.

After killing Eclipse I commented out everything in the file so it was just a Panel with a single 'Hello World' label in it.  Same problem.  I went so far as to download a fresh copy of Eclipse 2022-06 (I had been using and upgraded 2022-03).  Same problem.

I created a new file with a different class name, and copy/pasted the content from the original to it.  This file works fine.

I'd like to determine what went wrong and would like to know if there a less annoying fix than copy pasting source to a new file.

**Comment by wimjongman** [comment]
We need more information. What logs do you get? What file did you try to process? Maybe it is an old workspace issue?

**Reply by wimjongman** [reply]
By "old workspace issue" I mean that there could be old artifacts in the workspace. Please try the same swing project in a fresh workspace.
